website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -58,9 +58,8 @@ $ cd nginx-operator
 $ operator-sdk init --plugins=helm --domain=example.com
 ```
 
-Now that we have our new project initialized, we need to re-create each of our APIs. 
-Using our API example from earlier (`cache.example.com`), we'll use `cache` for the
-`--group` flag.
+**Note** Ensure that you use the same values for the flags to recreate the same Helm chart and API. If you have
+more than one chart or API you can add them via `operator-sdk create api` command. For further information check the [Helm Quickstart][quickstart]. 
```

Comment:
I think this change makes the doc less clear. I think we should revert this, and add an extra sentence about `--version` and `--kind`

```suggestion
Now that we have our new project initialized, we need to re-create each of our APIs.
Using our API example from earlier (`cache.example.com`), we'll use `cache` for the
`--group` flag.

For `--version` and `--kind`, we use `spec.versions[0].name` and `spec.names.kind`, respectively.
```